src/Raven.Client/Documents/Queries/IndexQuery.cs

```diff
@@ -151,6 +152,10 @@ public int PageSize
 
         public TimeSpan? WaitForNonStaleResultsTimeout { get; set; }
 
+        public (long? DocEtag, long? RefEtag)? CutoffEtag { get; set; }
+
+        public Stopwatch QueryDuration { get; set; }
+
```

Comment:
I'm not sure if these should be on the client side. 
Can this be moved to `IndexQueryServerSide` ?

Reply:
I haven't noticed it's in the client, I navigates to the base class, yes this can be moved up to the derived class.